skyportal/tests/frontend/test_sources.py

```diff
@@ -412,7 +412,7 @@ def test_centroid_plot(
 
         # Use this commented line to save a new version of the expected plot
         # if changes have been made to the component:
-        # generated_plot.save(expected_plot_path)
+        generated_plot.save(expected_plot_path)
```

Comment:
@acrellin, I had to un-comment these to get tests to pass, I wanted to call this out in case there's a better way to do this.

Reply:
I think this should just be temporarily un-commented while you run the tests locally, then commit the resulting saved file, and re-comment this line

Reply:
Ok, that's what I thought as well. I did that here:
351fc0e, and the tests failed.
This commit added them back in, tests passed: bb5b674
I commented them back out in the latest commit, and things are failing again on the same two tests: cbb784a

I don't believe these tests aren't connected to this work, so I'm not really sure what to do here. Any thoughts, @acrellin ?

Thanks for the help too, I do appreciate it.